lib/bridge/github/repo.rb

```diff
@@ -55,13 +56,17 @@ def details(data={})
         i
       end
 
+      template = issue_template
+      decoded_template = template ? Base64.decode64(template['content']) : nil
+
       {
         columns: columns,
         milestones: milestones,
         other_labels: other_labels.sort_by {|l| l['name'].downcase },
         link_labels: link_labels,
         assignees: assignees,
-        issues: issues
+        issues: issues,
+        issue_template: decoded_template
```

Comment:
Does Ruby support trailing commas on hashes and such? If so, I'd suggest we get in the habit of including them simply because it cleans up future diffs.